aerosonicdb/models/mfcc/mlp.py

```diff
@@ -199,42 +242,54 @@ def train_save_model(output_path=OUTPUT_PATH,
     print(class_weight_dict)
 
     # train model
-    model.fit(X_train, y_train, batch_size=batch_size, epochs=epochs, verbose=verbose, class_weight=class_weight_dict)
+    model.fit(
+        X_train,
+        y_train,
+        batch_size=batch_size,
+        epochs=epochs,
+        verbose=verbose,
+        class_weight=class_weight_dict,
+    )
 
     # save the model
-    model_path = os.path.join(output_path, filename, 'model')
+    model_path = os.path.join(output_path, filename, "model.keras")

```

Comment:
without the `.keras` extension this was crashing - keras now requires this extension apparently.